src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.tsx

```diff
@@ -1,10 +1,11 @@
 import { Box, Flex, Sans, Serif } from "@artsy/palette"
+import { LoosingBid } from "Assets/Icons/LoosingBid"
```

Comment:
typo `Losing` 🐞

Reply:
Oh English! https://www.google.com/search?q=How+do+you+spell+loosing+or+losing%3F&sa=X&ved=2ahUKEwiptIXmjJvfAhUSmuAKHeGxAAQQzmd6BAgGEAk&biw=1166&bih=921